_data/authors.yml

```diff
@@ -1,6 +1,6 @@
 # /_data/authors.yml
 
-Your_Name:
+Ankita_Sareen:
   name: "Ankita Sareen"
   bio: "Wanderlust with attempts to code!"
   avatar: "/assets/images/me.jpeg"
```

Comment:
Change the image name to your name. And make necessary changes below too. Keep the sample name existing as in https://github.com/khanfarhan10/khanfarhan10.github.io/blob/master/_data/authors.yml

I mean to say that keep this :
```Your_Name:
  name: "Your Awesome Name Here"
  bio: "Just Code Code Code? Nah, More Code."
  avatar: "/assets/images/ecopark.jpg"
  links:
    - label: "Email"
      icon: "fas fa-fw fa-envelope-square"
      url: "mailto:sampleperson@gmail.com"
    - label: "Website"
      icon: "fas fa-fw fa-link"
      url: "https://google.com"
    - label: "Twitter"
      icon: "fab fa-fw fa-twitter-square"
      url: "https://twitter.com/someone"
```
And add stuff that you want below that below that like :
```
Ankita_Sareen:
  name: "Ankita Sareen"
  bio: "Wanderlust with attempts to code!"
  avatar: "/assets/images/me.jpeg"
```